packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -170,6 +170,11 @@ export interface ISemanticColors extends ISemanticTextColors {
    */
   smallInputBorder: string;
 
+  /**
+   * The border of a small input control in its hovered state; e.g. checkbox border color on hover.
+   */
+  smallInputBorderHovered: string;
```

Comment:
These will need to be added to /office-ui-fabric-react/apps/theming-designer/src/components/SemanticSlots.tsx to update them in the theme designer as well since that's being used by designers now to get the latest slot values.

Reply:
It's probably a better approach to have `SemanticSlots.tsx` pull the theme object and as long as the app keeps its dependencies always latest you can iterate over the keys of the semanticColors property of the theme and build your array. This way you have the truth in one place and your app will always up to date. I don't think engineers know that in case they add a new semantic slot they need to update it the theme-designer app.

Reply:
@aneeshack4 and I have discussed this but it's difficult... Long story short she's working on it. We definitely agree that having the truth in one place is ideal.

Reply:
@Vitalius1 I've been wanting to do that but the data structure that makeSemantic returns isn't compatible because it doesn't return the palette values (like white or themePrimary etc), it returns the actual hex strings. @phkuo @marygans We might be able to reverse lookup the hex string into the palette that the theme object has to find the palette value right?

Reply:
That assumes the palette values are unique, which is true at least 99.99% of the time. You'll run into an edge case if the primary color overlaps with the neutrals, or if text and background color overlap (but in both those cases, you'll experience massive issues with contrast anyways). So yeah, I think that would work!

Reply:
I tried this approach out and unfortunately a bunch of the palette colors have the same hex value... for example, bodyBackground's hex value in the none variant is "#ffffff". We want the reverse lookup to get the palette string of "white" but it ends up grabbing "backgroundColor". Another example is accentButtonBackground whose hex value in the non variant is "#0078d4" and its correct palette value is "accent" but the reverse lookup grabs "primaryColor" instead because primaryColor's hex value is also "#0078d4". So using the reverse look up to find the palette string isn't really panning out....